### DISCUSSION TITLE
Civitai for LLMs

### DISCUSSION THREAD

**Discussion by NotSkynet** [civitai/civitai#427] [Ideas]
I propose the development of an intuitive and user-friendly platform (like Civitai) specifically designed for downloading and managing Large Language Models (LLMs). This platform will either be a standalone website or integrated as a separate section within Civitai.

**Comment by busssard** [comment]
If CivitAI made an effort into this, it could quickly evolve into a leading marketplace of AI Finetunes and even foundation models.

**Comment by Nickdev7** [comment]
have you started development?

**Reply by busssard** [reply]
> have you started development?

In the end huggingface is doing it

**Comment by sovos15** [comment]
Even just having an LLM with LLava to serve as a search option would be a powerful tool. I would pay to have it remember me and learn to act right. Apparently this could be done last year [by saving a clip embed of the images to reference. ](https://simonwillison.net/2023/Sep/12/llm-clip-and-chat/)

**Comment by SunGreen777** [comment]
This is a good idea and an opportunity to get more information about the model than what they give on Hug
